pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "xcodeproj"
-version = "2.0.1"
+version = "2.1.0"
```

Comment:
Should we bump the major version instead?

Reply:
I think that since it's a minor version we can get away with it. I'm not convinced this is used many people anyway.

Reply:
Just to point out, it won't install 2.1.0 if you are on an older Python version as it will be marked incompatible.